<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QColumnView" name="columnView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-9</y>
      <width>161</width>
      <height>641</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QColumnView{
background: #1c1c1c;
}</string>
    </property>
   </widget>
   <widget class="QColumnView" name="columnView_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>641</width>
      <height>631</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QColumnView{
background: #363636;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>511</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bebas Neue</family>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
</string>
    </property>
    <property name="text">
     <string>DATAbase controller</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRep1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>159</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bebas Neue</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #f8f000;
text-align: left;</string>
    </property>
    <property name="text">
     <string>Replicaciones</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>571</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QFrame" name="frame_10">
       <property name="styleSheet">
        <string notr="true">background: #787878;
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QFrame" name="frame_11">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>0</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #f8f000;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="btnBackup2">
         <property name="geometry">
          <rect>
           <x>-2</x>
           <y>0</y>
           <width>191</width>
           <height>32</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Bebas Neue</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background: #f8f000;
</string>
         </property>
         <property name="text">
          <string>BackUp</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>123</width>
          <height>72</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="styleSheet">
            <string notr="true">color: white;</string>
           </property>
           <property name="text">
            <string>Aquí se muestra:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_14">
           <property name="styleSheet">
            <string notr="true">color: white;</string>
           </property>
           <property name="text">
            <string>Hacer BackUP master</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_24">
           <property name="styleSheet">
            <string notr="true">color: white;</string>
           </property>
           <property name="text">
            <string>Lista de bases de datos</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_15">
           <property name="styleSheet">
            <string notr="true">color: white;</string>
           </property>
           <property name="text">
            <string>Lista de backups</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_12">
       <property name="styleSheet">
        <string notr="true">background: #787878;
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QFrame" name="frame_13">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>0</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #f24c3d;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="btnMon2">
         <property name="geometry">
          <rect>
           <x>-2</x>
           <y>0</y>
           <width>191</width>
           <height>32</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Bebas Neue</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Monitoreo</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="layoutWidget_3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>110</width>
          <height>72</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="label_16">
           <property name="styleSheet">
            <string notr="true">color: white;</string>
           </property>
           <property name="text">
            <string>Aquí se puede revisar:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_18">
           <property name="styleSheet">
            <string notr="true">color: white;</string>
           </property>
           <property name="text">
            <string>Estatus de la base</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_17">
           <property name="styleSheet">
            <string notr="true">color: white;</string>
           </property>
           <property name="text">
            <string>Usuarios conectadios</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_19">
           <property name="styleSheet">
            <string notr="true">color: white;</string>
           </property>
           <property name="text">
            <string>Y algo más</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_14">
       <property name="styleSheet">
        <string notr="true">background: #787878;
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QFrame" name="frame_15">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>0</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #38a6a6;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="btnRep2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>191</width>
           <height>32</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Bebas Neue</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Replicaciones</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="layoutWidget_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>144</width>
          <height>88</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="label_20">
           <property name="styleSheet">
            <string notr="true">color: white;</string>
           </property>
           <property name="text">
            <string>Aquí se observa:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_21">
           <property name="styleSheet">
            <string notr="true">color: white;</string>
           </property>
           <property name="text">
            <string>3 masters y 3 esclavos</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_22">
           <property name="styleSheet">
            <string notr="true">color: white;</string>
           </property>
           <property name="text">
            <string>2 hibridos y 2 esclavos</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_23">
           <property name="styleSheet">
            <string notr="true">color: white;</string>
           </property>
           <property name="text">
            <string>1 Master, 1 hibrido, 1 esclavo</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnMon1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>159</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bebas Neue</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #f8f000;
text-align: left;</string>
    </property>
    <property name="text">
     <string>Monitoreo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBackup1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>159</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bebas Neue</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #f8f000;
text-align: left;</string>
    </property>
    <property name="text">
     <string>BackUp</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>159</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bebas Neue</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #f8f000;
text-align: left;</string>
    </property>
    <property name="text">
     <string>HOME</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>590</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#787878;</string>
    </property>
    <property name="text">
     <string>Designed and coded by Secret eye antisocial club</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>571</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #787878;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>571</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #f27507;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bebas Neue</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Conectar al servidor:</string>
      </property>
     </widget>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>60</y>
       <width>171</width>
       <height>251</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bebas Neue</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>lista de servidores master:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bebas Neue</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>NOmbre del host o direccion ip:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtHost">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>100</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bebas Neue</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Usuario:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtUserLog">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>120</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtPswd">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>180</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;
color:white;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>160</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bebas Neue</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Contraseña:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConectar">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>220</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bebas Neue</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #f27507;
</string>
     </property>
     <property name="text">
      <string>conectar</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtConectado">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>250</y>
       <width>281</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;
</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red;</string>
    </property>
    <property name="text">
     <string>Conectar al servidor es para
crear un backup o ingresar al 
monitoreo de ese Master en 
especifico, para poder ver las 
replicaciones ve directo a 
&quot;Replicaiones&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bebas Neue</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:red;</string>
    </property>
    <property name="text">
     <string>AVISO!</string>
    </property>
   </widget>
   <zorder>columnView_2</zorder>
   <zorder>columnView</zorder>
   <zorder>label</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>btnRep1</zorder>
   <zorder>pushButton_8</zorder>
   <zorder>btnBackup1</zorder>
   <zorder>btnMon1</zorder>
   <zorder>label_2</zorder>
   <zorder>frame</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
